README.md

```diff
@@ -42,9 +42,9 @@ The entire execution engine was rebuild with performance in mind, in many cases
 - ✔ Proxies
 - ✔ Reflect
 - ✔ Symbols
-- ❌ Tail calls
+- ✔ Tail calls (_deprecated_)
```

Comment:
I don't think we can make this claim. This would special logic in runtime and we won't currently pass the test cases. Maybe we could just state `Won't be implemented`.